int16 bodyid
vector3f centerOfMass
joint[19] joints